Packs/CommonScripts/Scripts/GridFieldSetup/GridFieldSetup.py

```diff
@@ -0,0 +1,75 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+from typing import Dict, Any
+import traceback
+
+
+def grid_field_setup(keys: Optional[Any], vals: dict) -> List[Dict[str, Any]]:
+    """Returns a list of dictionaries based on the key/values provided.
+    :type keys: ``str``
+    :type vals: ``dict``
+    :param keys: comma separated values used for keys (columns)
+    :param vals: dictionary of the value assigned to keys
+    :return: list of dictionaries
+    """
+    if keys:
+        key_list = keys.split(',')
+    num_entries = len(key_list)
+    res_list = []
+    temp = {}
+    count = 0
+    while count < num_entries:
+        tkey = 'val' + str(count + 1)
+        temp[key_list[count]] = vals[tkey]
+        count += 1
```

Comment:
```suggestion
        count += 1
        temp[key_list[count]] = vals[f'val{count}']
```

Reply:
@esharf , why did you reverse the order?

Reply:
fixed in [switch to argToList](https://github.com/demisto/content/pull/21641/commits/3c29bbb2b836a8b3243fbeb9368bedea4d44399b)